releasenotes/source/master.rst

```diff
@@ -3,4 +3,3 @@ Current Series Release Notes
 ============================
 
 .. release-notes:: Release Notes
-    :branch: origin/master
```

Comment:
Does removing this mean that it just takes the current branch?

Reply:
yes. See https://docs.openstack.org/reno/latest/user/sphinxext.html#examples

Reply:
Nice!